packages/visx-grid/src/grids/GridPolar.tsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import cx from 'classnames';
+import { Group } from '@visx/group';
+import GridAngle from './GridAngle';
+import GridRadial from './GridRadial';
+
+import { CommonGridProps, GridScale } from '../types';
+
+export type GridPolarProps<Scale extends GridScale> = CommonGridProps & {
```

Comment:
I think that this needs to actually take 2 `Scale` generics to be applied as needed below since `Radial` and `Angle` won't necessarily have the same type.

```suggestion
export type GridPolarProps<AngleScale extends GridScale, RadialScale extends GridScale> = CommonGridProps & {
```